examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -4690,6 +4708,8 @@ endpoint 1 {
   }
 
   server cluster Actions {
+    emits event StateChanged;
+    emits event ActionFailed;
```

Comment:
@emargolis This seems wrong based on a previous comment you had, however it seems according to zap:

```
> jq '.endpointTypes[].clusters[] | select(.enabled == 1) | select(.side == "server") | {name: .name, events: [.events[]? | select(.included ==1) | .name]}' examples/all-clusters-app/all-clusters-common/all-clusters-app.zap | grep 'Actions' -A 4
  "name": "Actions",
  "events": [
    "StateChanged",
    "ActionFailed"
  ]
  ```
  
  I believe once this PR is in we may want to do another pass and figure out if the emitted event list is correct.

Reply:
It should be fixed in #25046